include/cassandra.h

```diff
@@ -2565,8 +2565,8 @@ cass_cluster_set_use_schema(CassCluster* cluster,
 /**
  * Enable/Disable retrieving hostnames for IP addresses using reverse IP lookup.
  *
- * This is useful for authentication (Kerberos) or encryption (SSL) services
- * that require a valid hostname for verification.
+ * @deprecated Do not use. Using reverse DNS lookup to verify the certificate
+ * does not protect against man-in-the-middle attacks.
```

Comment:
Perhaps include mention of CPP-942 inline in comments for additional context?

Although it doesn't look like there's much reference to CPP tickets in comments elsewhere in the source so maybe not.